idd/Energy+.idd.in

```diff
@@ -64671,6 +64671,208 @@ PlantEquipmentOperation:ComponentSetpoint,
        \key Cooling
        \key Dual
 
+PlantEquipmentOperation:ThermalEnergyStorage,
+       \memo Plant equipment operation scheme for simpler input to control thermal (ice)
+       \memo energy storage systems.  It replaces a host of setpoint managers with simple,
+       \memo single input values.  For more complex controls, use the ComponentSetpoint
+       \memo scheme.
+    \min-fields 7
+   A1, \field Name
+       \required-field
+       \reference ControlSchemeList
+   A2, \field On-Peak Schedule
+       \required-field
+       \type object-list
+       \object-list ScheduleNames
+   A3, \field Charging Availability Schedule
+       \required-field
+       \type object-list
+       \object-list ScheduleNames
+   N1, \field Non-Charging Chilled Water Temperature
+       \memo Single temperature for chiller outlet when not in cooling season
+       \memo or during on-peak cooling (discharge)
+       \type real
+       \units C
+   N2, \field Off-Peak Chilled Water Temperature
+       \memo Single temperature for chiller outlet when off-peak during cooling
+       \memo season (charging)
+   A4, \field Equipment 1 Object Type
+       \required-field
+       \type alpha
+   A5, \field Equipment 1 Name
```

Comment:
@mjwitte @Myoldmopar Yes, I will add \notes to the fields of the first group.  And yes, I will change the field names as you requested.  I will use the Component term as I think that is more standard.  So, this will read...
Component 1 Object Type
Component 1 Name
etc. when I finish.  Hopefully, that will address all of your concerns.